### ISSUE TITLE
Unexpected CodeApprove messages after reassigning a reviewer

### ISSUE THREAD

**Issue by mtlynch** [codeapprove/feedback#103]
I just ran into some confusing output from CodeApprove on this PR:

https://github.com/tiny-pilot/tinypilot/pull/1542

![image](https://github.com/codeapprove/feedback/assets/7783288/77acc883-089b-435d-a7d1-2c921b0c1712)

1. The author assigned the review to me.
2. I reassigned the review to a teammate (removed myself, added teammate in a single step).
3. CodeApprove commented `👀 @<author> it's your turn, please take a look`
4. CodeApprove commented `⏳ @<teammate> please review this Pull Request`

Step (3) was unexpected. I'm guessing it's because CodeApprove first received the event of me removing myself as a reviewer and then separately received the event of me adding my teammate.

It's not a big deal, but if it's possible to fix, it would eliminate some confusion for a not-so-uncommon workflow.

**Comment by samatcodeapprove** [comment]
@mtlynch yep I can see how that is confusing and I think you nailed the reason. When I first added attention sets I actually did debounce/batch reviewer actions on the frontend but I removed that because some people were closing the page immediately after adding reviewers which caused a bug.

Sounds like it's time for me to revisit this batching, probably on the backend this time.
